test/units/TEST-50-DISSECT.sysext.sh

```diff
@@ -1008,5 +1011,28 @@ if [[ -w /usr ]]; then
 fi
 run_sysext_tests "$FAKE_ROOTS_DIR"
 
+install_extension_images
+
+# Test that nested mountpoints are carried over into and back from the sysext overlayfs.
+ln -s /tmp/app0.raw /var/lib/extensions/app0.raw
+mkdir /tmp/foo
+mkdir /tmp/bar
+mkdir /usr/qed
```

Comment:
we try hard not to change the fs outside of working directories - can you do /var/tmp here or so instead?

Reply:
No the mountpoint has to be inside /usr, I'll use something that exists